example/components/Resistor/examples/SeriesExample.m

```diff
@@ -39,10 +39,10 @@ - (void)viewDidLoad {
 
 @end
 
-@implementation SeriesResistorExample (CatalogByConvention)
+@implementation SeriesExample (CatalogByConvention)
 
 + (NSArray<NSString *> *)catalogBreadcrumbs {
```

Comment:
Because this API isn't strongly enforced it may be reasonable to update the examples to just return an unspecialized NSArray.